pages/strategies.tsx

```diff
@@ -1,44 +1,314 @@
+      0n;
+
+    const borrowRewardsAPR =
+      rates['USDC']
+        ?.map((r) => (r.borrow * (ratio - WEI_PER_ETHER)) / WEI_PER_ETHER)
+        .reduce((acc, curr) => acc + curr, 0n) ?? 0n;
+
+    return toPercentage(Number(marketAPR + collateralRewardsAPR + borrowRewardsAPR) / 1e18);
+  }, [getMarketAccount, rates, usdcDepositAPR]);
+
+  const veloRate = useVELOPoolAPR() ?? '0%';
+  const extraRate = toPercentage(Number(useExtraDepositAPR() ?? 0) / 1e18);
+
+  const featured: (Strategy & { chainId?: number })[] = [
+    {
+      title: t('Maximize your yield'),
+      description: t('Amplify gains or mitigate risk with the power of leverage and deleverage in your investments.'),
+      tags: [
+        { prefix: t('up to'), text: `${maxYield} APR` },
+        { text: t('Advanced'), size: 'small' as const },
+      ],
+      button: (
+        <Button fullWidth variant="contained" onClick={() => startLeverager()} data-testid="leverage">
+          {t('Leverage')}
+        </Button>
+      ),
+      isNew: true,
+      source: 'exactly' as const,
+      imgPath: '/img/strategies/featured_leverage.svg',
+    },
+    {
+      title: t('Refinance Loans'),
+      description: t(
+        'Seamlessly transfer your debt positions between different pools or convert from fixed to variable rates, and vice versa.',
+      ),
+      tags: [
+        { prefix: t('FROM'), text: `${lowestBorrowAPR} APR` },
+        { text: t('Basic'), size: 'small' as const },
+      ],
+      button: (
+        <Button fullWidth variant="contained" onClick={() => startDebtManager({})} data-testid="rollover">
+          {t('Rollover')}
+        </Button>
+      ),
+      isNew: true,
+      source: 'exactly' as const,
+      imgPath: '/img/strategies/featured_rollover.svg',
+    },
+    {
+      chainId: optimism.id,
+      title: t('Deposit EXA on Extra Finance'),
+      description: t('Deposit EXA on Extra Finance and earn interest on it.'),
+      tags: [
+        { prefix: t('up to'), text: `${extraRate} APR` },
+        { text: t('Basic'), size: 'small' as const },
+      ],
+      button: (
+        <a href="https://app.extrafi.io/lend/EXA" target="_blank" rel="noreferrer noopener" style={{ width: '100%' }}>
+          <Button fullWidth variant="contained">
+            {t('Go to Extra Finance')}
+          </Button>
+        </a>
+      ),
+      source: 'third-party' as const,
+      imgPath: '/img/strategies/featured_extra.svg',
+    },
+    {
+      title: t('Reduce Exposure'),
+      description: t('Reduce your risk by decreasing your investment exposure and borrowing less.'),
+      tags: [
+        { prefix: t('Health Factor'), text: hfLabel },
+        { text: t('Advanced'), size: 'small' as const },
+      ],
+      button: (
+        <Button fullWidth variant="contained" onClick={() => startLeverager()}>
+          {t('Deleverage')}
+        </Button>
+      ),
+      isNew: true,
+      soruce: 'exactly' as const,
```

Comment:
maybe I'm wrong but I think we can use `as const` for the whole `featured` array